protos/manual_control/manual_control.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package mavsdk.rpc.manual_control;
+
+option java_package = "io.mavsdk.manual_control";
+option java_outer_classname = "ManualControlProto";
+
+service ManualControlService
+{
+     /*
+     * Send command to enable Joystick mode 
+     * Enabling Joystic mode causes to change into Position Control mode
+     * drone still needs to be armed
+     * Needs to check if the drone is not currently flying or in mission if so drone needs to be paused first 
+     */
+    rpc StartPositionControl (PositionControlRequest)  returns(PositionControlResponse) {}
+    rpc StartAltitudeControl (AltitudeControlRequest)  returns(AltitudeControlResponse) {}
+     /*
+     * Send command to command to the drone to move forwards or backwards 
+     */
+    rpc ManualControlInput (ManualControlRequest) returns (ManualControlResponse) {}
```

Comment:
This would  be stabilized mode, right? I'm not sure that we should support this because it's generally hard to fly stabilized unless you have a connection with quite low latency.

Reply:
is stabilized mode  same as position mode ?
how much means here low latency ?

Reply:
Stabilized is often called Manual but it's basically attitude control with direct throttle. The direct throttle part is what requires some skill and low latency.

I would, for now, not implement it, unless you actually require it. Would that make sense?

Reply:
Ok, I'm removing it for now.

Reply:
Oh, I was confused here actually. This is not to change to manual mode but actually setting the manual input :man_facepalming:. Ok I'll bring it back!